scully.scully-docs.config.ts

```diff
@@ -1,12 +1,14 @@
+import { ScullyConfig, setPluginConfig } from '@scullyio/scully';
+import { DisableAngular } from 'scully-plugin-disable-angular';
 import './demos/plugins/docs-link-update';
-import { setPluginConfig } from '@scullyio/scully';
 
 setPluginConfig('md', { enableSyntaxHighlighting: true });
 
-exports.config = {
+export const config: ScullyConfig = {
   projectRoot: './apps/scully-docs/src',
   projectName: 'scully-docs',
   outDir: './dist/static/doc-sites',
+  defaultPostRenderers: [DisableAngular],
```

Comment:
These will get overwritten by the `postRenderers` defined on the route level (`L15`). So this will not execute the DisableAngular plugin on the `/docs/:slug` routes.
See https://github.com/scullyio/scully/blob/23c8fcf740b15c8cb420c5dc1bc2ac7339edea83/libs/scully/src/lib/renderPlugins/executePlugins.ts#L12

Suggestion:

Instead of 

```
const handlers = [
    route.type,
    ...(route.postRenderers || scullyConfig.defaultPostRenderers)
  ].filter(Boolean);
```

we could use 

```
 const handlers = [
    route.type,
    ...(scullyConfig.defaultPostRenderers ?? []),
    ...(route.postRenderers ?? [])
  ].filter(Boolean);
```

which will just extend the handlers instead of choosing or the other.